### ISSUE TITLE
about the target label

### ISSUE THREAD

**Issue by Ichbinhippo** [dongyp13/Translation-Invariant-Attacks#5]
Hi,Yinpeng. a question. why do you set the prediction of inception_resnet_v2 as the target label? and the loss is computed by the (y,pred_logits). so is this means that you want to genarate a adv with target label as inception_resnet_v2 predicted rather than a adv that can fool all the 4 models you gave in the model? if it is convient, hope to get you reply? thx！

**Comment by dongyp13** [comment]
Note that the code is for **untargeted attack**, so we want to fool all the 4 models to give a prediction different from that of `inception_resnet_v2`.  I used the prediction of `inception_resnet_v2` as the ground-truth label for each image since this model gives the highest accuracy on the image set.

**Comment by qilong-zhang** [comment]
@dongyp13 Hi, I'm a little confused about this code
```python
        with slim.arg_scope(inception_resnet_v2.inception_resnet_v2_arg_scope()):
            logit, end_points = inception_resnet_v2.inception_resnet_v2(
                x_input, num_classes=num_classes, is_training=False)

        predicted_labels = tf.argmax(end_points['Predictions'], 1)
        y = tf.one_hot(predicted_labels, num_classes)
```
I don't know tf model well, I mean, what the difference between `tf.argmax(end_points['Predictions'], 1)` and `tf.argmax(logit, 1)`? 
Does they are same?
Thank you!

**Comment by dongyp13** [comment]
They are the same.

**Comment by qilong-zhang** [comment]
thanks!
